packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -1172,4 +1173,25 @@ export class RpcClient implements RpcClientInterface {
       method: 'GET',
     });
   }
+
+  /**
+   *
+   * @param reqs contains the kind (PVM) and kernel to generate the origination proof from
```

Comment:
might want to be consistent using "params" defined in function instead of "reqs" contains ...?